src/codeu/chat/common/ConversationHeader.java

```diff
@@ -75,6 +73,8 @@ public ConversationHeader read(InputStream in) throws IOException {
 	// a hashmap with user id as key and their usertype as value
 	private HashMap<Uuid, UserType> userAccessRoles = new HashMap<Uuid, UserType>();
 
+	public LinkedList<Bot> bots = new LinkedList<Bot>();
```

Comment:
Does this have to be public? Is there any reason you chose a `LinkedList` instead of some other data structure? Comments or JavaDoc would make this more obvious.